modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -262,10 +272,9 @@ export default {
         }
       }
       this.editing = this.items.find(item => item._id === id);
-      this.modified = false;
     },
     edited(value) {
```

Comment:
It doesn't look like this method is used anymore.

Reply:
? updateEditing is still called. The modified flag is consulted.

Reply:
Right. The `edited` method isn't, though.